pwsh-extension-dev.code-workspace

```diff
@@ -0,0 +1,493 @@
+          "!**/.vscode-test/**"
+        ],
+        "skipFiles": [
+          "<node_internals>/**",
+          "**/node_modules/**",
+          "**/.vscode-test/**",
+          "**/app/out/vs/**" //Skips Extension Host internals
+        ],
+        "presentation": {
+          "hidden": true
+        }
+      },
+      {
+        "name": "Launch-Temp",
+        "type": "extensionHost",
+        "request": "launch",
+        "env": {
+          "VSCODE_PARENT_SESSION_ID": "${command:GetVsCodeSessionId}",
+        },
+        "runtimeExecutable": "${execPath}",
+        "args": [
+          // Runs the extension in an empty temp profile that is automatically cleaned up after use
+          // Undocumented: https://github.com/microsoft/vscode-docs/issues/6220
+          "--profile-temp",
+          "--extensionDevelopmentPath=${workspaceFolder:Client}",
+          "${workspaceFolder:Client}/examples"
+        ],
+        "sourceMaps": true,
+        // This speeds up source map detection and makes smartStep work correctly
+        "outFiles": [
+          "${workspaceFolder:Client}/**/*.js",
+          "!**/node_modules/**",
+          "!**/.vscode-test/**"
+        ],
+        "skipFiles": [
+          "<node_internals>/**",
+          "**/node_modules/**",
+          "**/.vscode-test/**",
+          "**/app/out/vs/**" //Skips Extension Host internals
+        ],
+        "presentation": {
+          "hidden": true
+        }
+      },
+      {
+        "name": "Launch-Isolated",
+        "type": "extensionHost",
+        "request": "launch",
+        "env": {
+          "VSCODE_PARENT_SESSION_ID": "${command:GetVsCodeSessionId}",
+        },
+        "runtimeExecutable": "${execPath}",
+        "args": [
+          // Runs the extension in an empty temp profile that is automatically cleaned up after use
+          // Undocumented: https://github.com/microsoft/vscode-docs/issues/6220
+          "--profile=pwsh-debug",
+          "--extensionDevelopmentPath=${workspaceFolder:Client}",
+          "${workspaceFolder:Client}/examples"
+        ],
+        "sourceMaps": true,
+        // This speeds up source map detection and makes smartStep work correctly
+        "outFiles": [
+          "${workspaceFolder:Client}/**/*.js",
+          "!**/node_modules/**",
+          "!**/.vscode-test/**",
+        ],
+        "skipFiles": [
+          "<node_internals>/**",
+          "**/node_modules/**",
+          "**/.vscode-test/**",
+          "**/app/out/vs/**" //Skips Extension Host internals
+        ],
+        "presentation": {
+          "hidden": true
+        }
+      },
+      {
+        // https://code.visualstudio.com/docs/csharp/debugger-settings
+        "name": "Attach to Editor Services",
+        "type": "coreclr",
+        "request": "attach",
+        // Waits for the extension terminal to become available and gets the PID, saves having to enter it manually.
+        "processId": "${command:PowerShell.PickPSHostProcess}",
```

Comment:
Does it? Because I think this command is the old one that asks for a PID.

Reply:
Nope, another victim of cut and paste madness, good catch.